client/views/edit/edit.js

```diff
@@ -0,0 +1,71 @@
+import { Instances } from '/lib/common.js';
+
+function showEditAdvError(reason) {
+  if (typeof currentError !== 'undefined') {
```

Comment:
There's something funny going on with `currentError`'s scope. You aren't declaring that variable anywhere that I can find, and you're relying on it having a life longer than a single function call. Instead of doing that, try to find an appropriate place for that state to live. The [Session](https://docs.meteor.com/api/session.html) object might be a good place for this.

Reply:
When there has already been an error message and the user does something which results in an error again, the if condition removes the older error message. Removing that if condition and doing something erroneous again and again results in multiple error messages just like in the picture below.
![without](https://user-images.githubusercontent.com/17961929/42637706-7f43d1f6-8609-11e8-8eab-1f735582d1a8.png)
here I pressed **Create** button again and again which resulted in multiple error messages.
`currentError` is initially undefined hence it doesn't go inside the if condition for the first time.

Reply:
Don't remove the condition - find where `currentError` is getting defined and give it a better home, like the `Session` object I referenced above.

Reply:
Just realized that `currentError` is defined as a global variable in [package.json](https://github.com/berkmancenter/question_tool/blob/dev/package.json). It was used in other functions(add, combine, etc) in the same way as above for showing errors. Do we still need a Session variable now?

Reply:
Since you didn't create that, we can set it aside for now as something to be refactored in the future.